internal/services/loadbalancer/backend_address_pool_address_resource.go

```diff
@@ -204,6 +236,25 @@ func (r BackendAddressPoolAddressResource) Read() sdk.ResourceFunc {
 				}
 			}
 
+			var inboundNATRulePortMappingList []inboundNATRulePortMapping
+			if rules := backendAddress.LoadBalancerBackendAddressPropertiesFormat.InboundNatRulesPortMapping; rules != nil {
+				for _, rule := range *rules {
+					rulePortMapping := inboundNATRulePortMapping{}
+
+					if rule.InboundNatRuleName != nil {
+						rulePortMapping.Name = *rule.InboundNatRuleName
+					}
+					if rule.FrontendPort != nil {
+						rulePortMapping.FrontendPort = *rule.FrontendPort
+					}
+
+					if rule.BackendPort != nil {
+						rulePortMapping.BackendPort = *rule.BackendPort
+					}
+					inboundNATRulePortMappingList = append(inboundNATRulePortMappingList, rulePortMapping)
+				}
+				model.PortMapping = inboundNATRulePortMappingList
+			}
```

Comment:
Can we move it to the block in: https://github.com/hashicorp/terraform-provider-azurerm/pull/15097/files#diff-220a8106557720f7b503162b853b5dfde9cc2c5b8e330e501e2cc47b586f1f3aL197?

Reply:
can you be more specifically?